src/components/container/ReasonCard/ReasonCard.scss

```diff
@@ -0,0 +1,59 @@
+.ReasonCard {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 0px;
+  gap: 16px;
+  max-width: 356px;
+  font-style: normal;
+
+  &__icon {
+    position: relative;
+    width: 48px;
+    height: 48px;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+
+    &--background {
+      width: 100%;
+      height: 100%;
+
+      background-color: var(--color-selective-yellow);
+      border-radius: 16px;
+
+      position: absolute;
+      left: 0%;
+      right: 0%;
+      top: 0%;
+      bottom: 0%;
+    }
+    &--image {
```

Comment:
Not merge related and really pedantic, but there should be a newline after the `}` 🙈